frontend/src/static/css/style.scss

```diff
@@ -104,6 +104,47 @@ header{
             margin-bottom:1em;
             color:mui-color('grey', '700');
         }
+
+        .error-message-box {
+            padding: 15px;
+            border-radius: 4px;
+            background-color: #F8D7DA;
+            border-color: #F5C6CB;
+            color: #721C24;
+            margin-bottom: 15px;
+            text-align: left;
+        }
+        #error-message-box__close-button {
```

Comment:
We are using `scss`, can you look into how other CSS sections deal with parent and child labels? Like [this](https://github.com/reposense/RepoSense/blob/master/frontend/src/static/css/style.scss#L151-L188)